### ISSUE TITLE
<CsScore> #include "file.sco" returns stack pointer = 0

### ISSUE THREAD

**Issue by Nandos0804** [csound/csound#1880]
# Step to recreate the problem
-  Csd that invokes two file, an orchestra and a score
```
<CsoundSynthesizer>
    
    <CsOptions>
    -o test.wav -W -m 96 --m-amps=1
    </CsOptions>

    <CsInstruments>
        sr = 44100
        kr = 44100
        nchnls = 1
        0dbfs = 1

        giSine ftgen 0, 0, 2^10, 10, 1

        #include "prova.orc"

    </CsInstruments>
    
    <CsScore>
        #include "prova.sco"
        e
    </CsScore>

</CsoundSynthesizer>
```
- Orc file with code from poscil3 example
```
instr 1

krnd  randomh 40, 440, 1	; produce random values
ain   poscil3 .6, krnd, giSine
kline line    1, p3, 0    	; straight line
aL,aR pan2    ain, kline	; sent across image
      outs    aL, aR

endin
```
- Score from the same example
```
i1 0 10
e
```

- Alle the file are stored in the same dircetory (non admin path). From windows 10 terminal invoking 
```
csound prova.csd
```
returns 
```
UnifiedCSD:  .\prova.csd
rtaudio: PortAudio module enabled ...
using callback interface
rtmidi: PortMIDI module enabled
setting path from . to C:\Users\giuse\Desktop
sorting score ...
stack pointer = 0
```
# Tried without success
- giving full path for the #include statement 
- using new #includestr
- tried "\filename.sco" and ".\filename.sco"

The include in the orchestra is working flawlessly

**Comment by vlazzarini** [comment]
I can't reproduce it using csound 7.0 beta or 6.18.1

```
victor@firebird debug % csound prova.csd
UnifiedCSD:  prova.csd
rtaudio: PortAudio module enabled ...
using callback interface
rtmidi: PortMIDI module enabled
sorting score ...
stack pointer = 0
	... done
--Csound version 7.0 beta (double samples) May 14 2024
[commit: 37e9b38ce41f2d5fd8032c34b9c20fa3a078e3de]
libsndfile-1.2.2
graphics suppressed, ascii substituted
sr = 44100.0, kr = 44100.000, ksmps = 1
0dBFS level = 1.0, A4 tuning = 440.0
ftable 101:
ftable 101:	1024 points, scalemax 1.000
               .-''''''-._
            _-'           '.
          _-                '.
         -                    '_
       .'                       -
      -                          '_
    _'                             .
   .                                -
  -                                  '_
_'_____________________________________._______________________________________
                                        -                                     .
                                         -                                   -
                                          '_                               _'
                                            .                             .
                                             -                           -
                                              '.                       _'
                                                -_                    -
                                                  -_                .'
                                                    -_           _.'
                                                      '-._____..'
orch now loaded
audio buffered in 1024 sample-frame blocks
writing 2048-byte blks of shorts to test.wav (WAV)
SECTION 1:
B  0.000 .. 10.000 T 10.000 TT 10.000 M:    -4.44
Score finished in csoundPerform().
inactive allocs returned to freespace
end of score.		   overall amps:    -4.44
	   overall samples out of range:        0
0 errors in performance
1024 2048 sample blks of shorts written to test.wav (WAV)
```
This is a zip file with the sources I used: 
[prova.zip](https://github.com/csound/csound/files/15313284/prova.zip)

**Comment by vlazzarini** [comment]
The problem is the missing newline in the included files.

**Comment by Nandos0804** [comment]
I confirm that this solves the problem. But may i ask why is the missing CR+LF only affecting the score include and not the orchestra include?

**Comment by vlazzarini** [comment]
different code in each case. We'll try and find a solution to make this exit more gracefully with an error message.

**Comment by vlazzarini** [comment]
crash fixed in #1883
